sway-ir/src/instruction.rs

```diff
@@ -429,6 +433,15 @@ impl<'a> InstructionInserter<'a> {
         asm_val
     }
 
+    pub fn addrof(self, value: Value, span_md_idx: Option<MetadataIndex>) -> Value {
```

Comment:
I think it reads a bit weirdly.  `addr_of` is better.
```suggestion
    pub fn addr_of(self, value: Value, span_md_idx: Option<MetadataIndex>) -> Value {
```